source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -348,6 +350,12 @@ public LatestClientAndLatestServiceBuilder WithForcingClientProxyType(ForceClien
             return this;
         }
         
+        public LatestClientAndLatestServiceBuilder WithClientRpcObserver(IRpcObserver? clientRpcObserver)
```

Comment:
To be able to get the RPC observer into the HalibutRuntime builder, we had to push it in.

I did consider maybe making a more general purpose `With` method that allows someone access to the `HalibutRuntimeBuilder` of the client when it is being built.

I wasn't sure if now was the time.